Gemfile

```diff
@@ -75,16 +75,21 @@ group :development, :test do
   gem 'awesome_print'
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
   gem 'puma', '~> 3.12'
-  gem 'rspec-rails'
+
```

Comment:
I would keep rspec gems into test only environment if you don't need for other env.